src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs

```diff
@@ -24,48 +25,52 @@
     :text  (i18n/label :t/decline)}])
 
 (defn- swipeable
-  [{:keys [active-swipeable extra-fn notification]} child]
+  [{:keys [extra-fn notification]} child]
   (let [{:keys [id author message]}     notification
         {:keys [contact-request-state]} message
         {:keys [public-key]}            (rf/sub [:multiaccount/contact])
-        outgoing?                       (= public-key author)]
+        outgoing?                       (= public-key author)
+        accept                          (rn/use-callback
```

Comment:
As is the case in many other places, here we are just caching functions with `rn/use-callback` because the AC was written before our new standards with hooks were defined.